index.html

```diff
@@ -54,12 +54,12 @@ <h5 >I'm a Software Developer.</h5>
         <!-- main -->
         <section class="main">
             <div id="recent-work" class="recent-work">
-                <h3 class="title">My Recent Works</h3>
+                <h1 class="title">My Recent Works</h1>
```

Comment:
As the last reviewer mentioned, is important that on your website you only have one `h1` tag because this should represent the main heading/subject for the whole page and it also helps for search engine optimization 🤓 Please make the necessary changes to your other 12 `h1` tags so that you only have one 🙌